sdk/maps/Azure.Maps.Rendering/CHANGELOG.md

```diff
@@ -2,13 +2,9 @@
 
 ## 1.0.0-beta.2 (Unreleased)
 
-### Features Added
-
 ### Breaking Changes
 
-### Bugs Fixed
-
-### Other Changes
+- Deprecate Elevation tile enum
```

Comment:
Thanks for the heads up. Updated :)

Reply:
I'm still seeing the "Breaking Changes" header with the change.   As mentioned, since this change is between two betas, we'll want to move this to "Other Changes"

Reply:
Hi @jsquire , sorry, it must be some rebase error that I didn't notice. The Changelog is updated.